utils/deploy/envoy-notls.yaml

```diff
@@ -0,0 +1,269 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: envoy
+  name: envoy
+data:
+  envoy.yaml: |
+    static_resources:
+      clusters:
+        - name: authorino_wasm
```

Comment:
When this cluster is named `authorino` I am unable to reach it from within the shim so for now it is named `authorino_wasm` - todo: investigate why this is the case